src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,58 @@
+package controller;
+
+import java.util.List;
+
+import domain.result.WinningResult;
+import domain.deck.Deck;
+import domain.deck.DeckFactory;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.Players;
+import util.YesOrNo;
+import view.InputView;
+import view.OutputView;
+
+public class BlackJackGame {
+
+    private static final int FIRST_CARD_COUNT = 2;
+    private final Deck deck;
+
+    public BlackJackGame() {
+        deck = DeckFactory.getDeck();
+    }
+
+    public void firstDealOut(Dealer dealer, Players players) {
+        for (int i = 0; i < FIRST_CARD_COUNT; i++) {
+            dealer.draw(deck.dealOut());
+            players.draw(deck);
+        }
+    }
+
+    public void additionalDealOut(Dealer dealer, Players players) {
+        players.getPlayers()
```

Comment:
Players를 일급 컬렉션으로 분리한 만큼 getter를 제거해보면 어떨까요?

Reply:
위의 피드백과 약간 충돌하게 되는 사안인데요.

User내부에서 view에 가까운 기능들을 아래와 같이 바꾸게 된다면, Players에서 전체 결과값을 반환할 수 없게되고 결국 view에서 각 player에 접근하기 위해 getter를 사용하게 됩니다.
```java
public List<String> getDrawResult() {
    return cards.stream()
            .map(Card::getName)
            .collect(Collectors.toList());
}
```
players에서 만큼은 view적 기능을 구현하도록 하는게 맞을까요?

Reply:
```suggestion
        players.forEach(this::playersAdditionalDraw);
```
와 같이 사용할 수 있도록 해보면 어떨까요?